evi-flutter-example/lib/main.dart

```diff
@@ -164,20 +165,16 @@ class _MyHomePageState extends State<MyHomePage> {
   List<int> _audioInputBuffer = <int>[];
 
   // EVI sends back transcripts of both the user's speech and the assistants speech, along
-  // with an analysis of the emotional content of the speech. This method takes the JSON
+  // with an analysis of the emotional content of the speech. This method takes
   // of a message from EVI, parses it into a `ChatMessage` type and adds it to `messages` so
   // it can be displayed.
```

Comment:
```suggestion
  // with an analysis of the emotional content of the speech. This method takes
  // a message from EVI, creates a `ChatEntry` and adds it to `messages` so
  // it can be displayed.
```